src/app/controllers/bandAlbumsController.js

```diff
@@ -0,0 +1,29 @@
+export function bandAlbumsController($scope, $routeParams, $location, ApiService) {
+    $scope.bandAlbumsCtrl = this;
+
+    ApiService.getAlbums($routeParams.bandId)
+    .then(function(response) {
+        this.albums = response;
+    }.bind(this));
+
+    ApiService.getArtist($routeParams.bandId)
+    .then(function(response) {
```

Comment:
if the object that returns from service is an artist, you should name the param artist instead of response. Check this same thing in other calls that you do in this file and other controllers.